LayoutTests/TestExpectations

```diff
@@ -3809,6 +3809,7 @@ webgl/2.0.y/conformance2/extensions/ext-texture-norm16.html [ Pass ]
 webgl/2.0.y/conformance2/extensions/oes-draw-buffers-indexed.html [ Pass ]
 webgl/2.0.y/conformance2/extensions/promoted-extensions-in-shaders.html [ Pass ]
 webgl/2.0.y/conformance2/rendering/rasterizer-discard-and-implicit-clear.html [ Pass ]
+webgl/2.0.y/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html [ Pass ]
```

Comment:
I think this passes also because the test is not requiring the extension, and the extension being disabled.. We can perhaps fix this later

Reply:
Yeah, the "enable draft extensions" flag should be enabled for test runs.